src/environments/mock-environment.ts

```diff
@@ -35,6 +35,22 @@ export const environment: Partial<GlobalConfig> = {
       timePerMethod: {[RestRequestMethod.PATCH]: 3} as any // time in seconds
     }
   },
+  // Authority settings
```

Comment:
Same as above...this mock file includes comments that say "Authority" instead of "Authentication" on lines 38, 40 and 47.  Those should be updated, as these settings don't related to our Authority control features.